examples/rl/ddpg_pendulum.py

```diff
@@ -170,20 +169,10 @@ def record(self, obs_tuple):
 
         self.buffer_counter += 1
 
-    # We compute the loss and update parameters
-    def learn(self):
-        # Get sampling range
-        record_range = min(self.buffer_counter, self.buffer_capacity)
-        # Randomly sample indices
-        batch_indices = np.random.choice(record_range, self.batch_size)
-
-        # Convert to tensors
-        state_batch = tf.convert_to_tensor(self.state_buffer[batch_indices])
-        action_batch = tf.convert_to_tensor(self.action_buffer[batch_indices])
-        reward_batch = tf.convert_to_tensor(self.reward_buffer[batch_indices])
-        reward_batch = tf.cast(reward_batch, dtype=tf.float32)
-        next_state_batch = tf.convert_to_tensor(self.next_state_buffer[batch_indices])
-
+    @tf.function
```

Comment:
Please inline a comment to explain why you use a tf.function here (or otherwise mention it in the text)

Reply:
Fixed!